compiler/one-cmds/onelib/utils.py

```diff
@@ -231,6 +239,31 @@ def get_optimization_list(get_name=False):
     return opt_list
 
 
+def get_arg_parser(target: Optional[str], cmd: str) -> Optional[ArgumentParser]:
+    if not target:
+        return None
+
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    # for python module naming convention
+    target_name = target.replace('-', '_')
+    command_schema_path = dir_path + f'/../../backends/command/{cmd}/{target_name}.py'
```

Comment:
(optional) if there some kind of path provider then we can ask that provider for this kind of information.

Reply:
Yeah, that's good point. I'll revise that later.